src/utils/typed-array-utils.js

```diff
@@ -35,7 +35,7 @@ export function getGLTypeFromTypedArray(arrayOrType) {
 // Used to auto deduce gl parameter types
 
 /* eslint-disable complexity */
-export function getTypedArrayFromGLType(glType, {clamped = true} = {}) {
+export function getTypedArrayFromGLType(glType, {clamped = false} = {}) {
```

Comment:
When type is 'GL_UNSIGNED_BYTE', `readPixels` expects  `Uint8Array` , so setting this to false by default. Tested all examples and lessons, should I test anything else for this change?